src/auth-service/utils/create-user.js

```diff
@@ -85,6 +87,47 @@ const RoleModel = (tenant) => {
   }
 };
 
+async function deleteCollection(db, collectionPath, batchSize) {
+  const collectionRef = db.collection(collectionPath);
+  const query = collectionRef.orderBy('__name__').limit(batchSize);
+
+  return new Promise((resolve, reject) => {
+    deleteQueryBatch(db, query, batchSize, resolve, reject);
+  });
+}
+
+function deleteQueryBatch(db, query, batchSize, resolve, reject) {
+  query.get()
+    .then((snapshot) => {
+      // When there are no documents left, we are done
+      if (snapshot.size == 0) {
```

Comment:
encouraged to use "`===`" than the double "`==`" for such comparisons. @BenjaminSsempala

Reply:
Thanks @Baalmart I've updated this.